content/features/featuresDeepDive/materials/using/masterPBR.md

```diff
@@ -96,6 +96,49 @@ Once the conversion done, let's see the custom options available on this version
+
+- **metallicF0Factor**: This is the same as `specularFactor` from `KHR_materials_specular`. This parameter is multiplied with `metallicReflectanceColor` to control the intensity of specular reflections.
+- **metallicReflectanceColor**: This is the same as `specularColorFactor` from `KHR_materials_specular`. This parameter sets specular color at F0. As surface normals move toward F90, this value is interpolated to white.
+- **metallicReflectanceTexture**: This is a texture that holds values for `metallicReflectanceColor` in the RGB channels and values for `metallicF0Factor` in the Alpha channel. The texture components are multiplied with `metallicReflectanceColor` and `metallicF0Factor` when calculating the final specular contribution. This texture can be used as a single input for both `metallicReflectanceColor` and `metallicF0Factor` if authoring a `PBRMaterial` right in engine. However, if working with textures from a glTF file authored with the `KHR_material_specular` extension, this parameter acts as `specularTexture` as defined in the extension. This is because the extension defines `specularTexture` and `specularColorTexture` as two different textures with `specularTexture` holding the strength of specular reflections in the alpha channel. If working with `KHR_material_specular` textures, assign the texture defined as `specularTexture` to the `metallicReflectanceTexture` parameter and set the `useOnlyMetallicFromMetallicReflectanceTexture` parameter to true.
+- **useOnlyMetallicFromMetallicReflectanceTexture**: If this parameter is set to true, only the Alpha channel of `metallicReflectanceTexture` is used in lighting calculations. This parameter is important if using textures authored for the `KHR_material_specular` extension because if both `metallicReflectanceTexture` and `reflectanceTexture` are assigned textures, `metallicReflectanceTexture` will be used by default unless `useOnlyMetallicFromMetallicReflectanceTexture` is set to true. If the parameter is set to true, the RGB channels of the texture assigned to `reflectanceTexture` will be used to determine values for `metallicReflectanceColor`.
+- **reflectanceTexture**: This is the same as `specularColorTexture` from `KHR_materials_specular`. The color stored in the texture’s RGB channels is multiplied with `metallicReflectanceColor` and scaled by `metallicF0Factor` to produce the final specular contribution **only if** there is no texture assigned to `metallicReflectanceTexture` **or** `useOnlyMetallicFromMetallicReflectanceTexture` is set to true. Otherwise, any texture assigned to `metallicReflectanceTexture` will have its RGB channels used in place of this texture.
+
+To help illustrate how these parameters work together consider the following snippets of code which are all valid approaches to controlling specular reflections in a metallic-roughness material.

```

Comment:
I feel like we can have one or more playgrounds for this part? Is that easy to do?

Reply:
I can make some PGs for them. It would require some external files for the textures, but I can make a couple quick examples.

Reply:
I have made a PG showing three ways to set up specular as described in the section. Let me know what you think.

Reply:
Looks good. Maybe labels might help, but not required.